app/forms/ask_a_question_form.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+class AskAQuestionForm
+  include ActiveModel::Model
+  attr_accessor :name, :email, :subject, :message, :location_code, :context, :title
+
+  validates :name, :email, :message, presence: true
+  validates :email, email: true
+
+  def email_subject
+    "[PULFA] #{subject_string}"
+  end
+
+  def subject_string
+    return title if subject == "collection"
+    subject
+  end
+
+  def subject_options
+    [
+      ["This Collection", "collection"],
+      ["Reproductions & Photocopies", "reproduction"],
+      ["Rights & Permissions", "permission"],
+      ["Access", "access"],
+      ["Other", "how much"]
+    ]
+  end
+
+  def submit
+    ContactMailer.with(form: self).contact.deliver
+    @submitted = true
+    @name = ""
+    @email = ""
+    @message = ""
+  end
+
+  def submitted?
+    @submitted == true
+  end
+
+  def routed_mail_to
+    case location_code
+    when "rbsc", "lae", "mss", "rarebooks"
+      "rbsc@princeton.edu"
```

Comment:
question: Are we comfortable with these being hard-coded in the form rather than configured in a YAML file?

Reply:
I put them in there like this because YAML isn't great at multiple keys having the same value, I wasn't sure if the routing would stay simple, and I sort of like having all the functionality in this one file rather than jumping between two to figure out what goes where.